pkg/kubelet/server_test.go

```diff
@@ -501,6 +538,166 @@ func assertHealthFails(t *testing.T, httpURL string, expectedErrorCode int) {
 	}
 }
 
+type authTestCase struct {
+	Method string
+	Path   string
+}
+
+func TestAuthFilters(t *testing.T) {
+	fw := newServerTest()
+
+	// Manually test all the non-web-service handlers
+	testcases := []authTestCase{
+		{"GET", "/healthz"},
```

Comment:
Do we generate doc of any kind to walk instead of having to keep this list up to date?